src/components/middle/immedia/Immedia.scss

```diff
@@ -0,0 +1,72 @@
+.immedia-presence {
+  width: 100%;
+}
+
+.immedia-header {
+  min-height: 110px;
+  max-height: 180px;
+  background: transparent;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  width: 100%;
+}
+
+.immedia-background {
+  display: none;
+}
+
+.rows {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
```

Comment:
Since I was testing with many participants, I completely forgot how this would look with few: Bad.
![Screenshot from 2021-12-23 14-02-13](https://user-images.githubusercontent.com/14120807/147271486-bb3aa49c-b767-4ef4-8de9-8f2e2115b16a.png)

Let's revert this or use `flex-start`. Sorry for the noise!

Reply:
Agree, it's better using `llex-start`.